docs/contents/build-stages.md

```diff
@@ -0,0 +1,52 @@
+# Controller Build Stages
+
+We a number of service controller ***build stages***:
+
+* [PROPOSED](#proposed)
+* [PLANNED](#planned)
+* [BUILD](#build)
+* [RELEASED](#released)
+
+## PROPOSED
+
+The `PROPOSED` stage indicates that someone has expressed interest in
+supporting said service in ACK.
+
+There will be one or more Github Issues and/or a Github Project for tracking
+the build of the ACK service controller for the service.
+
+The GitHub Issues **WILL NOT** be associated with a GitHub Milestone.
+
+## PLANNED
+
+The `PLANNED` status indicates that **the AWS service is on our radar** for
```

Comment:
"The `PLANNED` status indicates that we plan to make a controller for this service available in ACK."